src/display.ts

```diff
@@ -250,16 +251,25 @@ const drawFeature = (svg: SVGSVGElement, face: Face, info: FeatureInfo) => {
   }
 };
 
+/**
+ * Since spaces are illegal in HTML element IDs, we can use
+ * a string with spaces to convey that we want something else
+ * than getting a DOM element by ID.
+ */
+export const EXPORT_AS_SVG_STRING = "export as svg string";
```

Comment:
This is hacky, would be better to add a parameter to `display`. But is this even needed? You could maybe instead pass in an instance of `svgDocument` as `container`, and then wherever you're passing it in from would still have access to that instance.

That would also have the advantage of not including `svgDocument.js` in the bundle for `display.js`, since for current uses of faces.js svgDocument is not needed and it's just extra code included. There could be a separate function `exportAsString.js` that people can import for this use case.

Reply:
Good point. I have reverted my changes to display.ts and injected the pseudo DOM element in commit ce2e8725bebade72489e6beff50026a592e95205